### PR TITLE
Auto-advance date of the `next` tag

### PR DESCRIPTION
**Pull request by ddbeck**
Removes softprops/action-gh-release in favor of the `gh` CLI.

Fixes https://github.com/web-platform-dx/web-features/issues/2068

Closes https://github.com/web-platform-dx/web-features/pull/2425

### CODE REVIEW

File: .github/workflows/publish_next_web-features.yml [hunk 1]
```diff
@@ -68,20 +68,33 @@ jobs:
           VERSION: ${{ steps.version.outputs.VERSION }}
           TIMESTAMP: ${{ steps.timestamp_and_hash.outputs.TIMESTAMP }}
           SHORT_HASH: ${{ steps.timestamp_and_hash.outputs.SHORT_HASH }}
-      - run: npm publish --tag ${{ env.dist_tag }}
+      - if: ${{ env.NODE_AUTH_TOKEN }}
```
Comment (ddbeck) [inline .github/workflows/publish_next_web-features.yml:71]: I added this condition to make it easier to test on my fork.

**Comment by ddbeck** [comment]
@Elchi3 would you be interested in reviewing this? It's primarily a maintainer quality-of-life thing, to stop dismissing the Dependabot upgrade for the GitHub Action. I've tested it on my fork, so I've got fairly high confidence on this one.

**Review by Elchi3 — APPROVED** [review]

**Comment by szepeviktor** [comment]
@ddbeck Hello!

I am watching releases.
Now I get a notification on every commit to main.
Please advise.

**Comment by ddbeck** [comment]
Oh that sounds very annoying. Sorry about that, @szepeviktor—I'll open a PR to revert this until I find another workaround.

**Comment by szepeviktor** [comment]
Thank you.

**Comment by ddbeck** [comment]
I've opened https://github.com/web-platform-dx/web-features/pull/2453.

**Pull request merged**